test/parallel/test-child-process-fork-dgram.js

```diff
@@ -66,13 +78,16 @@ if (process.argv[2] === 'child') {
     sendMessages();
   });
 
+  var iterations = 0;
   var sendMessages = function() {
     var timer = setInterval(function() {
+      iterations++;
+
       client.send(
         msg,
         0,
         msg.length,
-        server.address().port,
+        serverPort,
```

Comment:
Is there a reason to add the serverPort variable? It only seems to be used here, and makes things a bit more complicated.